lib/test/fixtures/getDataContractFixture.js

```diff
@@ -0,0 +1,64 @@
+const DataContract = require('@dashevo/dpp/lib/dataContract/DataContract');
```

Comment:
Can we use the fixture from DPP?

Reply:
Yes, if we modify it in the way, that we can pass contractId to it.

Reply:
I see. Let's keep it then :)